importer/helpers/metadagbuilder.go

```diff
@@ -21,6 +21,22 @@ type MetaDagBuilderHelper struct {
 	metaDagRoot ipld.Node // Metadata Dag root node
 }
 
+func NewMetaDagBuilderHelper(idb DagBuilderHelper, spl chunker.Splitter, mroot ipld.Node) *MetaDagBuilderHelper {
+	return &MetaDagBuilderHelper{
+		db:          idb,
+		metaSpl:     spl,
+		metaDagRoot: mroot,
+	}
+}
+
+func GetSuperMeta(chunkSize uint64, maxLinks uint64, trickleFormat bool) interface{} {
```

Comment:
Why is it returning an `interface{}`? Should it be `SuperMeta` type? This looks unused right now.

Reply:
Done.